mono_repo/lib/src/package_config.dart

```diff
@@ -337,13 +338,13 @@ class CIJob implements HasStageName {
   List get _items => [description, package, sdk, stageName, tasks];
 }
 
-@JsonSerializable(includeIfNull: false)
+@JsonSerializable(includeIfNull: false, explicitToJson: true)
 class Task {
   static const _oldToNewTaskNames = {
     'dartfmt': 'format',
     'dartanalyzer': 'analyze',
   };
-  static const _tasks = {'format', 'analyze', 'test', 'command'};
+  static const _tasks = {'format', 'analyze', 'test', 'command', 'action'};
```

Comment:
I wonder if we should call this `github_action` instead of `action`? It might be a little more clear what it is

Reply:
Good thinking! Done 👍 sorry for the delay